internal/handlers/pg/msg_find.go

```diff
@@ -70,6 +70,17 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 		return nil, err
 	}
 
+	maxTimeMS, err := common.GetPositiveNumber(document, "maxTimeMS")
+	if err != nil {
+		return nil, err
+	}
+
+	runCtx, runCancel := context.WithTimeout(ctx, time.Duration(maxTimeMS)*time.Millisecond)
```

Comment:
`run` looks out of place in this code. I think it was copied from some `run` function. It is a bad name

Reply:
Renamed